test/integration/rocks/test_rocks.py

```diff
@@ -54,7 +54,12 @@ def test_rocks_module(tt_cmd, tmpdir):
             [tt_cmd, "rocks", "unpack", rock_file],
             cwd=tmpdir, env=dict(os.environ, PWD=tmpdir))
     assert rc == 0
-    rock_dir = rock_file.split('.', 1)[0]
+    rock_dir = ""
+    rock_dir_items = rock_file.split('.')
+    for i in range((len(rock_dir_items) - 2)):
```

Comment:
What is the name of the rock file and extracted directory? Why do we need this loop?

Reply:
Because previously rock_dir has been wrong if path contained "." in it.
E.g "foo.bar/tempdir/test/rock"